credhub.yml

```diff
@@ -0,0 +1,107 @@
+# Update with local file location
+- type: replace
+  path: /releases/-
+  value:
+    name: credhub
+    url: file:///tmp/credhub-0.3.1.tgz
+
+# Update dev_key with chosen value
+- type: replace
+  path: /instance_groups/name=bosh/jobs/-
+  value:
+    name: credhub
+    release: credhub
+    properties:
+      credhub: 
+        port: 8844
+        user_management:
+          uaa:
+            url: "https://((internal_ip)):8443"
+            verification_key: ((uaa_jwt_signing_key.public_key))
+        data_storage:
+          type: postgres
+          host: 127.0.0.1
+          port: 5432
+          username: postgres
+          password: ((postgres_password))
+          database: credhub
+        tls: ((credhub-tls))
+        log_level: info
+        encryption:
+          provider: dev_internal
+          dev_key: ABCDEF0123456789ABCDEF0123456789 #Replace with chosen value - 32 hex characters 
```

Comment:
Switching to a password-derived key in our backlog, but not coming in the near term.